### PR TITLE
Add Type to Query Model for Advanced Search Type

### PR DESCRIPTION
**Pull request by deevezina**

### CODE REVIEW

File: src/Vasont.Inspire.Models.csproj [hunk 1]
```diff
@@ -20,15 +20,15 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>CCMS</PackageTags>
      <PackageReleaseNotes>Added SubmissionAttributeModel</PackageReleaseNotes>
-    <Version>1.2.60</Version>
+    <Version>1.2.62</Version>
```
Comment (zeroadam) [inline src/Vasont.Inspire.Models.csproj:23]: Did we skip v 61?

File: src/Vasont.Inspire.Models.csproj [hunk 2]
```diff
@@ -20,15 +20,15 @@
...
     <NeutralLanguage>en</NeutralLanguage>
     <Copyright>Copyright (c) GlobalLink Vasont. All rights reserved.</Copyright>
   </PropertyGroup>
 
   <PropertyGroup>
     <BuildDocFx Condition="$(TargetFramework) != 'netstandard2.1' OR '$(Configuration)' != 'Release'">false</BuildDocFx>
-    <AssemblyVersion>1.2.60.0</AssemblyVersion>
-    <FileVersion>1.2.60.0</FileVersion>
+    <AssemblyVersion>1.2.62.0</AssemblyVersion>
+    <FileVersion>1.2.62.0</FileVersion>
```
Comment (zeroadam) [inline src/Vasont.Inspire.Models.csproj:31]: skip version?

**Review by zeroadam — APPROVED** [review]
see comments, otherwise looks good

File: src/Vasont.Inspire.Models.csproj (hunk 1 above)
Comment (deevezina) [reply src/Vasont.Inspire.Models.csproj:23]: this is weird because i had 61 prior

Comment (deevezina) [reply src/Vasont.Inspire.Models.csproj:23]: i'll just go to 62

**Pull request merged**